.github/workflows/CI.yml

```diff
@@ -1,6 +1,6 @@
 on:
   push:
-    branches: [master]
+    # branches: [master]
```

Comment:
I see, we should we leave it like this then ? :point_up:

Reply:
Nop, fixed it